build/charts/theia/values.yaml

```diff
@@ -131,6 +131,8 @@ clickhouse:
     # Specification to install a default ZooKeeper cluster. This is ignored
     # if zookeeperHosts are provided. 
     installZookeeper:
+      # -- Determine whether to enable persistent volume for zookeeper
+      createPersistentVolume: false
```

Comment:
How about using the same options in ch and grafana to provide PV support for local/nfs, and maybe also hostpath?

Reply:
I've tested the Zookeeper with PV. And it works fine when we have the scenario in the description. Two PVs will be created.
<img width="1467" alt="image" src="https://github.com/antrea-io/theia/assets/59460118/7440147f-8178-4daa-973e-d3d93bed030e">
Do we still need to create PV and StorageClass?

Reply:
Thank for reviewing! I've updated the options to match those we used for ClickHouse and Grafana.